File: data/EXP01/go/2348/2348.output_2.go
func zeroFilledSubarray(nums []int) int64 {
    count := int64(0)
    length := len(nums)
    for i := 0; i < length; i++ {
        if nums[i] == 0 {
            count++
        }
        for j := i + 1; j < length; j++ {
            if nums[j] == 0 {
                count++
            } else {
                break
            }
        }
    }
    return count
}

File: data/EXP01/python/2348/2348.output_2.py
# @lc app=leetcode id=2348 lang=python3
from typing import List

class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:    
        subarray_count = 0
        zeros_length = 0
        
        for num in nums:
            if num != 0:
                zeros_length = 0
            else:
                zeros_length += 1
                subarray_count += zeros_length * (zeros_length + 1) // 2
        
        return subarray_count